File: examples/codegen-result/src/typescript/api/resp/ServiceResponse.ts

/**
 * 服务响应对象
**/

export interface  ServiceResponse<T> {

        /**
            *属性说明：响应数据code，示例输入：
            *在java中的类型为：Integer
        **/
        code?: number;
        /**
            *属性说明：响应数据，示例输入：
        **/
        data?: T;
        /**
            *属性说明：响应数据消息，示例输入：
            *在java中的类型为：String
        **/
        message?: string;
}

File: examples/swagger-3/src/test/resources/asserts/java/retrofit/src/com/wuxp/codegen/swagger3/resp/ServiceResponse.java
package com.wuxp.codegen.swagger3.resp;

import lombok.Data;

import javax.validation.constraints.NotNull;




@Data
public class  ServiceResponse<T> {

          /**
              *字段在java中的类型为：String
          **/
         String message;
          /**
              *字段在java中的类型为：Integer
          **/
         Integer code;
         T data;
}
